drivers/chibios/ws2812.c

```diff
@@ -22,6 +22,14 @@
 #    define WS2812_OUTPUT_MODE PAL_MODE_OUTPUT_OPENDRAIN
 #endif
 
+// WS2812 Byte Order
+#define WS2812_BYTE_ORDER_RGB 0
+#define WS2812_BYTE_ORDER_GRB 1
+
+#ifndef WS2812_BYTE_ORDER
+#    define WS2812_BYTE_ORDER WS2812_BYTE_ORDER_GRB
+#endif
+
```

Comment:
color.h is included by ws2812.h, so there should be no need to add these in the ChibiOS .c files here.
I'd also prefer to see it in ws2812.h instead of color.h, but I can't see how that would work with the AVR implementations, seeing as they appear to rely on the ordering of the struct. Speaking of which, might as well add the byte order checking to `cRGBW` too, just in case :)

Reply:
@fauxpark to my knowledge the only RGBW LED used or available is the SK6812RGBW - which will use the RGB data struct

https://cdn-shop.adafruit.com/product-files/2757/p2757_SK6812RGBW_REV01.pdf

Reply:
Hence "just in case" - you never know.

Reply:
The ergodox ez uses RGBW,  and sparkfun has RGBW strips.  

So yes, please include that.